backend-rust/README.md

```diff
@@ -17,24 +17,83 @@ To run the services, the following dependencies are required to be available on
 
 ## Run the Indexer Service
 
-For instructions how to use the indexer run:
+For instructions on how to use the indexer run:
 
 ```
-ccdscan-indexer --help
+cargo run --bin ccdscan-indexer -- --help
 ```
 
+Example:
+
+```
+cargo run --bin ccdscan-indexer -- --node http://localhost:20001 --max-parallel-block-preprocessors 20 --max-processing-batch 20
+cargo run --bin ccdscan-indexer -- --node http://node.testnet.concordium.com:20000
+cargo run --bin ccdscan-indexer -- --node https://grpc.testnet.concordium.com:20000
+```
+
+Note: Queries like `getSourceModule` might timeout (are disabled) on our public-facing nodes. The recommendation is to run
```

Comment:
The goal is to prevent each developer from discovering this problem again when running against the public nodes. Let's see if we get a good explanation from SRE and revise the comment if needed.